orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -0,0 +1,731 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import copy
+import onnx
+import os
+import torch
+import numpy as np
+import threading
+
+import torch
```

Comment:
use pytest? 

see orttraining_test_ortmodule_api.py for reference?

Reply:
Yeah, fair.

Reply:
move all single GPU test for pytest; distributed test case remain.